app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -566,18 +566,17 @@ export const getSimulatedBlockNumber = async (ethereumClientService: EthereumCli
 	return await ethereumClientService.getBlockNumber()
 }
 
-export const getSimulatedTransactionByHash = async (ethereumClientService: EthereumClientService, simulationState: SimulationState | undefined, hash: bigint): Promise<EthereumSignedTransactionWithBlockData|undefined> => {
+export const getSimulatedTransactionByHash = async (ethereumClientService: EthereumClientService, simulationState: SimulationState | undefined, hash: bigint): Promise<EthereumSignedTransactionWithBlockData | undefined> => {
 	// try to see if the transaction is in our queue
 	if (simulationState === undefined) return await ethereumClientService.getTransactionByHash(hash)
 	for (const [index, simulatedTransaction] of simulationState.simulatedTransactions.entries()) {
 		if (hash === simulatedTransaction.signedTransaction.hash) {
-			const v = 'v' in simulatedTransaction.signedTransaction ? simulatedTransaction.signedTransaction.v : (simulatedTransaction.signedTransaction.yParity === 'even' ? 0n : 1n)
 			const additionalParams = {
 				blockHash: getHashOfSimulatedBlock(),
 				blockNumber: await ethereumClientService.getBlockNumber(),
 				transactionIndex: BigInt(index),
 				data: simulatedTransaction.signedTransaction.input,
-				v : v,
+				signature: 378356n, // signature field to fix ethers. TODO, remove when ethers is fixed
```

Comment:
This constant should probably be in hex, and having it be 65 bytes would also help add clarity to what exactly it is.

Also, where is this constant from?  Does 65 0 bytes not work?

Reply:
its "ethers"
3=e
7=t
8=h
5=r
6=s